wweval/R/eval_fit.R

```diff
@@ -26,7 +26,14 @@ eval_fit_ww <- function(config_index,
   wwinference::create_dir(output_dir)
   wwinference::create_dir(raw_output_dir)
 
-  params <- wwinference::get_params(params_path) |> as.data.frame()
+  # params <- wwinference::get_params(params_path) #nolint
+  # Temporarily get parameters from local package install to test, since
+  # need updates from branch of wwinferece
```

Comment:
Not following this comment, you may already have a checklist for this; if not, I suggest adding a separate issue for additional fixes needed after this is merged.

Reply:
Good idea -- I will make a checklist. Basically we need to change the `params.toml` so it has all the same variables the `wwinference` package now expects

Reply:
Added to the PR description